OpenAPI spec has wrong examples

### System Info

**Info**
` docker run -p 8080:80 ghcr.io/huggingface/text-embeddings-inference:cpu-1.8`
`curl 127.0.0.1:8080/info | jq
{
  "model_id": "BAAI/bge-large-en-v1.5",
  "model_sha": null,
  "model_dtype": "float32",
  "model_type": {
    "embedding": {
      "pooling": "cls"
    }
  },
  "max_concurrent_requests": 512,
  "max_input_length": 512,
  "max_batch_tokens": 16384,
  "max_batch_requests": 8,
  "max_client_batch_size": 32,
  "auto_truncate": false,
  "tokenization_workers": 16,
  "version": "1.8.3",
  "sha": "78502d8e61223d2c73d4bb7aeaea46787e90d596",
  "docker_label": "sha-78502d8"
}`

### Information

- [x] Docker
- [x] The CLI directly

### Tasks

- [x] An officially supported command
- [ ] My own modifications

### Reproduction

1. Open http://localhost:8080/docs/#/Text%20Embeddings%20Inference/embed
2. Press "Try it out" and get an example:
```json
{
  "dimensions": null,
  "inputs": "string",
  "normalize": true,
  "prompt_name": "null",
  "truncate": false,
  "truncation_direction": "right"
}
```
3.  "Execute"
4. Get error code 422:
```text
Failed to deserialize the JSON body into the target type: truncation_direction: unknown variant `right`, expected `Left` or `Right` at line 7 column 33
```


### Expected behavior

Example should have correct values for  `prompt_name` and `truncation_direction`:
```json
{
  "dimensions": null,
  "inputs": "string",
  "normalize": true,
  "prompt_name": null,
  "truncate": false,
  "truncation_direction": "Right"
}
```

Also ran spec through https://github.com/stoplightio/spectral with openAPI ruleset and got this back:
```text
spectral lint https://huggingface.github.io/text-embeddings-inference/openapi.json

https://huggingface.github.io/text-embeddings-inference/openapi.json
   1:1    warning  oas3-api-servers           OpenAPI "servers" must be present and non-empty array.
  17:14   warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./decode.post
  53:33     error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./decode.post.responses[400].content.application/json.example.error_type
  67:33     error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./decode.post.responses[413].content.application/json.example.error_type
  81:33     error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./decode.post.responses[422].content.application/json.example.error_type
  90:14   warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./embed.post
  126:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./embed.post.responses[400].content.application/json.example.error_type
  140:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./embed.post.responses[413].content.application/json.example.error_type
  154:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./embed.post.responses[422].content.application/json.example.error_type
  168:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./embed.post.responses[424].content.application/json.example.error_type
  182:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./embed.post.responses[429].content.application/json.example.error_type
  228:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./embed_all.post.responses[400].content.application/json.example.error_type
  242:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./embed_all.post.responses[413].content.application/json.example.error_type
  256:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./embed_all.post.responses[422].content.application/json.example.error_type
  270:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./embed_all.post.responses[424].content.application/json.example.error_type
  284:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./embed_all.post.responses[429].content.application/json.example.error_type
  293:14  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./embed_sparse.post
  329:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./embed_sparse.post.responses[400].content.application/json.example.error_type
  343:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./embed_sparse.post.responses[413].content.application/json.example.error_type
  357:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./embed_sparse.post.responses[422].content.application/json.example.error_type
  371:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./embed_sparse.post.responses[424].content.application/json.example.error_type
  385:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./embed_sparse.post.responses[429].content.application/json.example.error_type
  394:13  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./health.get
  413:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Unhealthy"?   paths./health.get.responses[503].content.application/json.example.error_type
  422:13  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./info.get
  443:13  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./metrics.get
  464:14  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./predict.post
  500:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./predict.post.responses[400].content.application/json.example.error_type
  514:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./predict.post.responses[413].content.application/json.example.error_type
  528:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./predict.post.responses[422].content.application/json.example.error_type
  542:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./predict.post.responses[424].content.application/json.example.error_type
  556:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./predict.post.responses[429].content.application/json.example.error_type
  602:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./rerank.post.responses[400].content.application/json.example.error_type
  616:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./rerank.post.responses[413].content.application/json.example.error_type
  630:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./rerank.post.responses[422].content.application/json.example.error_type
  644:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./rerank.post.responses[424].content.application/json.example.error_type
  658:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./rerank.post.responses[429].content.application/json.example.error_type
  667:14  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./similarity.post
  703:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./similarity.post.responses[400].content.application/json.example.error_type
  717:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./similarity.post.responses[413].content.application/json.example.error_type
  731:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./similarity.post.responses[422].content.application/json.example.error_type
  745:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Backend"?     paths./similarity.post.responses[424].content.application/json.example.error_type
  759:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Overloaded"?  paths./similarity.post.responses[429].content.application/json.example.error_type
  768:14  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./tokenize.post
  804:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Empty"?       paths./tokenize.post.responses[400].content.application/json.example.error_type
  818:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Validation"?  paths./tokenize.post.responses[413].content.application/json.example.error_type
  832:33    error  oas3-valid-media-example   "error_type" property must be equal to one of the allowed values: "Unhealthy", "Backend", "Overloaded", "Validation", "Tokenizer", "Empty". Did you mean "Tokenizer"?   paths./tokenize.post.responses[422].content.application/json.example.error_type
  841:14  warning  operation-description      Operation "description" must be present and non-empty string.                                                                                                           paths./v1/embeddings.post
  875:27    error  oas3-valid-media-example   "example" property must have required property "code"                                                                                                                   paths./v1/embeddings.post.responses[400].content.application/json.example
  889:27    error  oas3-valid-media-example   "example" property must have required property "code"                                                                                                                   paths./v1/embeddings.post.responses[413].content.application/json.example
  903:27    error  oas3-valid-media-example   "example" property must have required property "code"                                                                                                                   paths./v1/embeddings.post.responses[422].content.application/json.example
  917:27    error  oas3-valid-media-example   "example" property must have required property "code"                                                                                                                   paths./v1/embeddings.post.responses[424].content.application/json.example
  931:27    error  oas3-valid-media-example   "example" property must have required property "code"                                                                                                                   paths./v1/embeddings.post.responses[429].content.application/json.example
  983:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.DecodeRequest.properties.skip_special_tokens.default
  984:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.DecodeRequest.properties.skip_special_tokens.example
 1015:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.EmbedAllRequest.properties.truncate.default
 1016:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.EmbedAllRequest.properties.truncate.example
 1060:24    error  oas3-valid-schema-example  "default" property type must be integer,null                                                                                                                            components.schemas.EmbedRequest.properties.dimensions.default
 1061:24    error  oas3-valid-schema-example  "example" property type must be integer,null                                                                                                                            components.schemas.EmbedRequest.properties.dimensions.example
 1070:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.EmbedRequest.properties.normalize.default
 1071:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.EmbedRequest.properties.normalize.example
 1082:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.EmbedRequest.properties.truncate.default
 1083:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.EmbedRequest.properties.truncate.example
 1131:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.EmbedSparseRequest.properties.truncate.default
 1132:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.EmbedSparseRequest.properties.truncate.example
 1238:24    error  oas3-valid-schema-example  "default" property type must be integer,null                                                                                                                            components.schemas.Info.properties.max_batch_requests.default
 1239:24    error  oas3-valid-schema-example  "example" property type must be integer,null                                                                                                                            components.schemas.Info.properties.max_batch_requests.example
 1245:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Info.properties.max_batch_tokens.example
 1250:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Info.properties.max_client_batch_size.example
 1256:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Info.properties.max_concurrent_requests.example
 1261:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Info.properties.max_input_length.example
 1288:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Info.properties.tokenization_workers.example
 1399:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.OpenAICompatEmbedding.properties.index.example
 1437:24    error  oas3-valid-schema-example  "default" property type must be integer,null                                                                                                                            components.schemas.OpenAICompatRequest.properties.dimensions.default
 1438:24    error  oas3-valid-schema-example  "example" property type must be integer,null                                                                                                                            components.schemas.OpenAICompatRequest.properties.dimensions.example
 1502:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.OpenAICompatUsage.properties.prompt_tokens.example
 1507:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.OpenAICompatUsage.properties.total_tokens.example
 1568:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.PredictRequest.properties.raw_scores.default
 1569:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.PredictRequest.properties.raw_scores.example
 1573:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.PredictRequest.properties.truncate.default
 1574:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.PredictRequest.properties.truncate.example
 1620:24    error  oas3-valid-schema-example  "example" property type must be number                                                                                                                                  components.schemas.Prediction.properties.score.example
 1633:24    error  oas3-valid-schema-example  "example" property type must be integer                                                                                                                                 components.schemas.Rank.properties.index.example
 1639:24    error  oas3-valid-schema-example  "example" property type must be number                                                                                                                                  components.schemas.Rank.properties.score.example
 1662:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.RerankRequest.properties.raw_scores.default
 1663:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.RerankRequest.properties.raw_scores.example
 1667:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.RerankRequest.properties.return_text.default
 1668:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.RerankRequest.properties.return_text.example
 1681:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.RerankRequest.properties.truncate.default
 1682:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.RerankRequest.properties.truncate.example
 1737:24    error  oas3-valid-schema-example  "default" property type must be boolean,null                                                                                                                            components.schemas.SimilarityParameters.properties.truncate.default
 1738:24    error  oas3-valid-schema-example  "example" property type must be boolean,null                                                                                                                            components.schemas.SimilarityParameters.properties.truncate.example
 1799:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.SimpleToken.properties.special.example
 1857:24    error  oas3-valid-schema-example  "default" property type must be boolean                                                                                                                                 components.schemas.TokenizeRequest.properties.add_special_tokens.default
 1858:24    error  oas3-valid-schema-example  "example" property type must be boolean                                                                                                                                 components.schemas.TokenizeRequest.properties.add_special_tokens.example
```

--- a/router/src/http/types.rs
+++ b/router/src/http/types.rs
@@ -194,9 +194,10 @@ impl<'__s> ToSchema<'__s> for PredictInput {
 }
 
 #[derive(Debug, Clone, Copy, PartialEq, Deserialize, ToSchema, Eq, Default)]
-#[serde(rename_all = "lowercase")]
 pub(crate) enum TruncationDirection {
+    #[serde(alias = "left", alias = "Left")]
     Left,
+    #[serde(alias = "right", alias = "Right")]
     #[default]
     Right,
 }

